### PR TITLE
refactor: remove mui from faq section (accordion)

### PR DESCRIPTION
**Pull request by Zackydout101**
Fixed styling and removed mui

### CODE REVIEW

**Review by arcanistzed — CHANGES_REQUESTED** [review]
There should be an icon for each item that changes from a `+` to a `-`. Perhaps a cool transition would be to have one of the lines fade while the other rotates 90 degrees, with similar code to the hamburger menu.

There should also be an animation for the expansion and contraction, rather than have it be instant.

File: src/components/FAQ/FAQ.module.css [hunk 1]
```diff
@@ -20,6 +20,7 @@
 	font-family: Coolvetica;
```
[the remark this answers is not in the document]
Comment (Zackydout101) [reply src/components/FAQ/FAQ.module.css:20]: In the old code we use this .header {
	font-family: Coolvetica; we don't use a css variable font

[the remark this answers is not in the document]
Comment (arcanistzed) [reply src/components/FAQ/FAQ.module.css:20]: Yeah, there are still quite a few places that use that. We need to be consistent though. Some of the fonts use CSS variables. Probably a good idea to make another PR which switches them to all be the same.

**Comment by arcanistzed** [comment]
Is it possible to make it animate when it's collapsed and when it expands after the first time?

**Comment by arcanistzed** [comment]
I would recommend using the `<details>` and `<summary>` HTML elements for accessibility

**Pull request closed without merging**